packages/contracts/src/template/ChaosBurnSys.sol

```diff
@@ -1,32 +1,59 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.8.21;
 
+import "forge-std/console2.sol";
+
 import {System} from "@latticexyz/world/src/System.sol";
 import {Config, ConfigData} from "../codegen/index.sol";
 import {Player, PlayerData} from "../codegen/index.sol";
+import {Game, GameData} from "../codegen/index.sol";
 import {PlayerStatus} from "../codegen/common.sol";
+import {IWorld} from "src/codegen/world/IWorld.sol";
+import {SystemSwitch} from "@latticexyz/world-modules/src/utils/SystemSwitch.sol";
 
 /**
  * @notice this system make the amount of token into a random things.
  */
 contract ChaosBurnSys is System {
+    mapping(uint256 => bool) public hasBurnedThisRound;
```

Comment:
it doesn't work at all. Should add valut in `mud.config.ts`

Reply:
I think it's best not to write like this. In mud, storage defination and logic is seperate, we'd better obey this rule

Reply:
> Hmmmm... I see. Then, do you have any idea to implement this?: I just want to limit players to change burn_amount once per round.
> 
> Or, I wonder how can I add new components(storage) for the new system.

If we implment this demand, we need a dedicated `mud.config.ts`. I prefer not to implement it in this repo.

Reply:
If a player want to add new storage, he can fork the repo and modify `mud.config.ts`. Once his proposal is passed, the new `mud.config.ts` can be merged to the the game session repo.